bin/download_schemas.sh

```diff
@@ -0,0 +1,100 @@
+      exit 1
+    fi
+    #Remove key '$schema' of the common file because it would overwrite the same key in the main file
+    jq 'del(."$schema")' umm-cmn-json-schema-temp.json > umm-cmn-json-schema.json
+    #Merge and create js file
+    echo "$(echo "const ummCSchema =")" "$(jq -s '.[0] * .[1]' umm-c-json-schema.json umm-cmn-json-schema.json)" > ummCSchema-temp.js
+    echo "$(echo "export default ummCSchema")" >> ummCSchema-temp.js
+    #Replace pointers to the common file
+    sed 's/umm-cmn-json-schema.json#/#/g' ummCSchema-temp.js > ummCSchema.js
+}
+
+ummS() {
+    echo "Downloading ummS"
+    #Reads version number
+    schema_version=`jq '.ummVersions.ummS' ../static.config.json`
+    schema_version=${schema_version//\"/}
+    #Download
+    curl https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/raw/service/v${schema_version}/umm-s-json-schema.json > umm-s-json-schema-temp.json
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-s-json-schema.json"
+      exit 1
+    fi
+    jq --arg ummJsonSchemaUrl "$ummJsonSchemaUrl" '."$schema" = $ummJsonSchemaUrl' umm-s-json-schema-temp.json > umm-s-json-schema.json
+    #Create js file
+    echo "$(echo "const ummSSchema =")" "$(jq '.' umm-s-json-schema.json)" > ummSSchema.js
+    echo "$(echo "export default ummSSchema")" >> ummSSchema.js
+}
+
+ummV() {
+    echo "Downloading ummV"
+    #Reads version number
+    schema_version=`jq '.ummVersions.ummV' ../static.config.json`
+    schema_version=${schema_version//\"/}
+    #Download
+    curl https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/raw/variable/v${schema_version}/umm-var-json-schema.json > umm-var-json-schema-temp.json
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-var-json-schema.json"
+      exit 1
+    fi
+    jq --arg ummJsonSchemaUrl "$ummJsonSchemaUrl" '."$schema" = $ummJsonSchemaUrl' umm-var-json-schema-temp.json > umm-var-json-schema.json
+    #Create js file
+    echo "$(echo "const ummVarSchema =")" "$(jq '.' umm-var-json-schema.json)" > ummVarSchema.js
+    echo "$(echo "export default ummVarSchema")" >> ummVarSchema.js
+}
+
+ummT() {
+    echo "Downloading ummT"
+    #Reads version number
+    schema_version=`jq '.ummVersions.ummT' ../static.config.json`
+    schema_version=${schema_version//\"/}
+    #Download
+    curl https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/raw/tool/v${schema_version}/umm-t-json-schema.json > umm-t-json-schema-temp.json
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-t-json-schema.json"
+      exit 1
+    fi
+    jq --arg ummJsonSchemaUrl "$ummJsonSchemaUrl" '."$schema" = $ummJsonSchemaUrl' umm-t-json-schema-temp.json > umm-t-json-schema.json
+    #Create js file
+    echo "$(echo "const ummTSchema =")" "$(jq '.' umm-t-json-schema.json)" > ummTSchema.js
+    echo "$(echo "export default ummTSchema")" >> ummTSchema.js
+}
+
+cleanup() {
+  rm ./*-temp.*
```

Comment:
I wonder if you should put a \ in front of it, depending on the shell env, it may prompt "are you sure?"

Reply:
`\rm ./*-temp.*`

Reply:
Done